core/commands/urlstore.go

```diff
@@ -101,10 +107,9 @@ time.
 		}
 
 		chk := chunk.NewSizeSplitter(hres.Body, chunk.DefaultBlockSize)
-		prefix := cid.NewPrefixV1(cid.DagProtobuf, mh.SHA2_256)
 		dbp := &ihelper.DagBuilderParams{
 			Dagserv:    n.DAG,
-			RawLeaves:  true,
+			RawLeaves:  false,
```

Comment:
Maybe we should only flip this for v0, so the v1 path is not changed (false: `UnixFS` node, rather than `raw`).